app/src/components/main/modal/darkModalLayout/darkModalLayout.scss

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.container {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.wrapper {
+  position: absolute;
+  top: 165px;
+  right: 0;
+  bottom: 0;
+  left: 56px;
+  display: flex;
+  justify-content: center;
+  padding-bottom: 100px;
+  background-color: rgba($COLOR--dark-modal, 0.95);
+  overflow: hidden;
+}
+
+.layout {
+  width: 100%;
+  max-width: 920px;
+  margin: 30px 15px 15px 15px;
+}
+
+:global {
+  .window-animation-enter {
+    bottom: 1000px;
+    padding-bottom: 0;
+  }
+  .window-animation-enter-active {
+    bottom: 0;
+    transition: all 300ms linear;
+  }
+}
```

Comment:
I would change the animation, here's a good example http://reactcommunity.org/react-transition-group/css-transition